### PR TITLE
Add documentation for TimeZone permission

### PR DESCRIPTION
**Pull request by haptikrajasashtikar**
- We require the timezone of the user to greet the user appropriately
- In case of FB we get this permission from User Profile API
- Timezone requires permission to be accessed
- This change highlights how can we request that permission

CIA-6078

### CODE REVIEW

File: docs/other/facebook.md [hunk 1]
```diff
@@ -169,9 +169,29 @@ For certain permissions, we need verfications from Facebook. There are two compo
       - Refer to this Facebook Documentation to learn more - [Submitting your Messenger Bot](https://developers.facebook.com/docs/messenger-platform/app-review/).
 
       - > When you share your bot for permission, there is a testing freeze. No one can test the bot on the facebook page.
+ 
+  2. pages_user_timezone permission:
+
+      - Timezone of a user is required to greet the user appropriately
+          - For ex: If the user greets the bot "Good Morning", the bot will greet back the user based in user's timezone
```
Comment (medhaanand) [inline docs/other/facebook.md:176]:
```suggestion
          - For ex: If the user greets the bot "Good Morning", the bot will greet back the user based on user's timezone 
```

File: docs/other/facebook.md [hunk 2]
```diff
@@ -169,9 +169,29 @@ For certain permissions, we need verfications from Facebook. There are two compo
...
+      - This permission needs to be requested on Page Level as well
+          - Go to client's Facebook Page -> Settings -> Advanced Messaging -> Under Info About People -> Select the "Time Zone" field -> click the "Request" button
+       
+        ![Page Level Permission](assets/FB-PAGE-LEVEL-PERMISSIONS.jpeg)    
+
+      - Facebook Messenger review team manually verifies your bot and gives permission accordingly.
+
+      - Refer to this Facebook Documentation to learn more - [Submitting your Messenger Bot](https://developers.facebook.com/docs/messenger-platform/app-review/).
+
+      - > When you share your bot for permission, there is a testing freeze. No one can test the bot on the facebook page.
```
Comment (medhaanand) [inline docs/other/facebook.md:192]: Delete extra line's space here @haptikrajasashtikar

File: docs/other/facebook.md (hunk 1 above)
Comment (tejas-haptik) [reply docs/other/facebook.md:176]: @haptikrajasashtikar showcasing his grammar in PR and gets corrected
Screen Shot 2020-05-28 at 4 39 30 PM

Meanwhile @haptikrajasashtikar team members..... @RandiveSushant @dikshit24 @tejas-haptik 
Screen Shot 2020-05-28 at 4 48 38 PM

Comment (haptikrajasashtikar) [reply docs/other/facebook.md:176]: Co-incidentally I am also dancing with my team mates.....
😆

Comment (tejas-haptik) [reply docs/other/facebook.md:176]: @haptikrajasashtikar "co-incidentally", this is also you right now
Screen Shot 2020-05-28 at 5 03 59 PM

Comment (haptikrajasashtikar) [reply docs/other/facebook.md:176]: 😆

**Pull request merged**